tests/lib/rules/no-implicit-globals.js

```diff
@@ -1241,6 +1326,231 @@ ruleTester.run("no-implicit-globals", rule, {
                     type: "VariableDeclarator"
                 }
             ]
+        },
+
+        //------------------------------------------------------------------------------
+        // exported
+        //------------------------------------------------------------------------------
+
```

Comment:
Can you add a test to confirm that `exported` does not allow variables that are implicitly created by assignments ([global variable leaks](https://eslint.org/docs/latest/rules/no-implicit-globals#global-variable-leaks)). For example, `/* exported foo */ foo = 1;` should still be invalid for this rule.

Reply:
done! [0c32702](https://github.com/eslint/eslint/pull/16343/commits/0c32702e56b4c60ee8d166e66a5fa4dbe5a9ee7f)